.github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md

```diff
@@ -0,0 +1,136 @@
+
+## Graviton-based self-hosted github action worker
+
+### Step 1. Create an EC2 instance with Graviton
+
+Create an AWS Graviton-based EC2 instance. For example,
+
+```
+# or download from https://github.com/aws/aws-k8s-tester/releases
+cd ${HOME}/go/src/github.com/aws/aws-k8s-tester
+go install -v ./cmd/ec2-utils
+
+# create arm64 AL2 instance
+AWS_K8S_TESTER_EC2_ON_FAILURE_DELETE=true \
+AWS_K8S_TESTER_EC2_LOG_COLOR=true \
+AWS_K8S_TESTER_EC2_REGION=us-west-2 \
+AWS_K8S_TESTER_EC2_S3_BUCKET_CREATE=true \
+AWS_K8S_TESTER_EC2_S3_BUCKET_CREATE_KEEP=true \
+AWS_K8S_TESTER_EC2_REMOTE_ACCESS_KEY_CREATE=true \
+AWS_K8S_TESTER_EC2_ASGS_FETCH_LOGS=false \
+AWS_K8S_TESTER_EC2_ASGS='{"GetRef.Name-arm64-al2-cpu":{"name":"GetRef.Name-arm64-al2-cpu","remote-access-user-name":"ec2-user","ami-type":"AL2_arm_64","image-id-ssm-parameter":"/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-arm64-gp2","instance-types":["m6g.xlarge"],"volume-size":40,"asg-min-size":1,"asg-max-size":1,"asg-desired-capacity":1}}' \
+AWS_K8S_TESTER_EC2_ROLE_CREATE=true \
+AWS_K8S_TESTER_EC2_VPC_CREATE=true \
+ec2-utils create instances --enable-prompt=true --auto-path
+```
+
+### Step 2. Install github action on the host
+
+SSH into the instance, and install the github action self-hosted runner (see [install scripts](https://github.com/etcd-io/etcd/settings/actions/runners/new?arch=arm64&os=linux)).
```

Comment:
I cannot open the link (404). Please double check / document the permissions needed.

Reply:
Just gave you the admin access. Can you check again?

Reply:
@ptabor Just added you to `cncf-admins`. Now you should have access to the settings.